pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "Canary"
+version = "1.0.4"
+description = ""
+license = "GPL-3.0-only"
+repository = "https://github.com/idoneam/Canary"
+readme = "README.md"
+authors = []
+
+[tool.poetry.dependencies]
+python = "^3.6.0"
+beautifulsoup4 = "==4.8.0"
+sympy = "==1.5"
+requests = ">=2.20.0"
+tabulate = "==0.8.6"
+"discord.py" = "==1.2.5"
```

Comment:
Is there a reason why `discord.py` is enclosed in quotes?

Reply:
As mentioned on discord, poetry doesn't like dots. Enclosing the dependency name forces `discord.py` to be read as a dependency as opposed to a property.

Reply:
Cool, thanks for explaining!